pandas/tests/frame/test_indexing.py

```diff
@@ -542,6 +541,16 @@ def test_setitem_boolean(self):
         np.putmask(expected.values, mask.values, df.values * 2)
         assert_frame_equal(df, expected)
 
+    def test_setitem_boolean_ndarary(self):
```

Comment:
can you parametrize to pass both ``lambda mask: mask, lambda mask: mask.value`` (IOW pass a dataframe and the ndarray)

Reply:
can you do this

Reply:
You saw my comment? You definitely want to use parametrize rather than just adding it like I did below?

https://github.com/dhirschfeld/pandas/blob/520adf778867e9054ee3634e23654eadfc0507ab/pandas/tests/frame/test_indexing.py#L557-L560

Reply:
yes

Reply:
can you parametrize this